kconfig-lib.sh

```diff
@@ -1,20 +1,21 @@
+#!/bin/bash
```

Comment:
I think this file is only meant to be sourced, not run so I wouldn't do this.

shellcheck is like checkpatch: it's right most of the time but all the time.

There are ways to silence specific warnings but since shellcheck is not run in CI in this repo then I wouldn't bother.